### ISSUE TITLE
There is an issue with starting 

### ISSUE THREAD

**Issue by jamesbaker1** [futurice/pepperoni-app-kit#207]
![simulator screen shot jun 19 2017 10 01 56 pm](https://user-images.githubusercontent.com/26907599/27317394-fd911f60-553a-11e7-89fa-a6d915b1a688.png)

**Comment by tatva-nisarg** [comment]
Any update on this?

**Comment by mmchougule** [comment]
I am seeing the same issue.

**Comment by Unforgiven-wanda** [comment]
Yes, you must have updated your `react-navigation` dependency, its code has changed slightly.
You can either use the [upgraded-dependencies](https://github.com/futurice/pepperoni-app-kit/tree/maintenance/upgrade-dependencies) branch, with this issue fixed, or fix it yourself in your app, just take a look at [this](https://github.com/futurice/pepperoni-app-kit/pull/202/files) specifically at the `counterView` and `colorView` files, you'll see the change you have to make

**Comment by tatvamac** [comment]
![simulator screen shot 04-jul-2017 11 38 28 am](https://user-images.githubusercontent.com/29597672/27817168-571318e0-60ad-11e7-8503-ae70ef22059d.png)

getting the warning and always seems Activity indicator after above solution from [this](https://github.com/futurice/pepperoni-app-kit/pull/202/files) specifically at the counterView and colorView file.

**Comment by xap5xap** [comment]
I followed all the steps in the Getting started document and the setup guide. Sadly, the app didn't start. Same error as other people. I wanted this project to save me some time but after spending some time trying to fix all the errors, I gave up. This projects is a good idea but it doesn't run.

**Comment by kusti8** [comment]
It is a known issue now that react-navigation changed its parameters. I've addressed it, short term, in #212

**Comment by dominictracey** [comment]
It's pretty straightforward to fix one's app if you look at kusti8's [PR](https://github.com/futurice/pepperoni-app-kit/pull/202/files)

**Comment by saenz** [comment]
rolling back the r-n version fixed it for me: npm install react-navigation@1.0.0-beta.7

**Comment by anton-aleksandrov** [comment]
`yarn install`
